pkg/projects/resource_project.go

```diff
@@ -123,52 +125,72 @@ func projectResource() *schema.Resource {
 			Default:     false,
 			Description: "Alerts will be sent when reaching 75% and 95% of the storage quota. Serves as a notification only and is not a blocker",
 		},
+
+		"member": {
+			Type:     schema.TypeSet,
+			Optional: true,
+			Elem: &schema.Resource{
+				Schema: map[string]*schema.Schema{
+					"name": {
+						Type:             schema.TypeString,
+						Required:         true,
+						ValidateDiagFunc: validation.ToDiagFunc(validation.StringIsNotEmpty),
+					},
+					"roles": {
+						Type:     schema.TypeSet,
+						Required: true,
+						Elem:     &schema.Schema{Type: schema.TypeString},
+					},
+				},
+			},
+		},
 	}
 
-	var unpackProject = func(data *schema.ResourceData) (interface{}, string, error) {
+	var unpackProject = func(data *schema.ResourceData) (string, Project, Users, error) {
 		d := &ResourceData{data}
 
 		project := Project{
 			Key:                    d.getString("key", false),
 			DisplayName:            d.getString("display_name", false),
 			Description:            d.getString("description", false),
-			StorageQuota:           GigabytlesToBytes(d.getInt("max_storage_in_gigabytes", false)),
+			StorageQuota:           GibabytesToBytes(d.getInt("max_storage_in_gibabytes", false)),
```

Comment:
I really feel like an implementation in the state function is the way to go.

Reply:
Good point